src/scripts/main.js

```diff
@@ -1,3 +1,20 @@
 'use strict';
 
-// write your code here
+function copyTabCol() {
+  const table = document.querySelector('table');
```

Comment:
Consider checking if the `table` element exists before proceeding. If `document.querySelector('table')` returns `null`, attempting to call `querySelectorAll` on it will result in a runtime error.